### DISCUSSION TITLE
When is dialogic 2 official release scheduled?

### DISCUSSION THREAD

**Discussion by lethefrost** [dialogic-godot/dialogic#2244] [Q&A] [answered]
Thank you so much for this amazing addon (and for being really helpful and active in the community)! Just wondering if there is a particular reason why it is not in the assetlib? And if the release date is already scheduled? Can't wait to see the full 2.0 version.

**Comment by CakeVR** [comment, marked as answer]
Let me start with the AssetLib: Dialogic 2 will be released on the AssetLib once it gets its official `2.0.0` release.

When will this happen? The only answer here is: When the time has come. There are milestones to look at but even they may change still.
There will be a new Alpha 14 soon that fixes up many bugs and brings some new features. Feel free to join the Discord if you want to get a better feel on the development.

https://github.com/dialogic-godot/dialogic/milestone/9
https://github.com/dialogic-godot/dialogic/milestone/8

**Reply by lethefrost** [reply]
Thank you so much for confirming! Looking forward to seeing it soon. ☺️

**Reply by CakeVR** [reply]
Same, I'm looking forward to it as well. You can also try out the latest changes on the `main` branch.

Oh, and please select the post as your answer, I cannot do that, but it marks the thread as answered for others browsing around :)

**Reply by lethefrost** [reply]
I'd really like to help! But I cannot find the way to set the answer - can you please help me find some interactive elements in my interface (as shown in the screenshot) that will set this post as the answer?
<img width="670" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/47198693/e6e81ce2-b052-4fa6-ba33-9dc2a7848cfb">

I tried to search for the [GitHub official document](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) regarding how to do this, but in their diagram there seems to be a button ![](https://docs.github.com/assets/cb-84373/mw-1440/images/help/discussions/comment-mark-as-answer-button.webp) and they have the check mark icon with black background. Should I close this thread first and then will be able to select an answer?

**Reply by lethefrost** [reply]
I've tried to close it but it still doesn't work...

**Reply by CakeVR** [reply]
Hey, I just figured it out! I changed it to a Q&A and now I was able to set the answer myself. Makes sense! Thanks for your efforts!

**Reply by lethefrost** [reply]
Thanks for your efforts too! so glad to see the alpha 14 release today. Very impressive plugin and developers!
